marketplace/optimizely/src/AppPage/index.js

```diff
@@ -26,6 +41,12 @@ const styles = {
   light: css({
     opacity: '0.6',
     marginTop: tokens.spacingM
+  }),
+  logo: css({
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: tokens.spacingXl
```

Comment:
I guess `marginBottom` would also be nice so the logo doesn't touch the browser window if more CTs are selected.